src/javascript/binary/static/profittable/profittable.ui.js

```diff
@@ -0,0 +1,100 @@
+
+var ProfitTableUI = (function(){
+    "use strict";
+
+    var profitTableID = "profit-table";
+    var cols = ["buy-date", "ref", "contract", "buy-price", "sell-date", "sell-price", "pl"];
+    var header = ["Purchase Date", "Ref.", "Contract", "Purchase Price", "Sale Date", "Sale Price", "Profit/Loss"];
```

Comment:
better to make it like this `var header = [Content.localize().textPurchaseDate` as explained earlier also, we should have centralized texts for new javascripts. Old js is scattered we should not proceed with same way as old one